src/main/apiClient.ts

```diff
@@ -93,7 +93,7 @@ export type ApiClientConfig = {
   language?: string
   scope?: string
   sso: boolean
-  pkceEnabled?: boolean
+  pkceEnforced?: boolean
```

Comment:
The value should always be present in theory. The `?` was probably used when the feature was first integrated.